account/accounts.go

```diff
@@ -62,7 +62,14 @@ type RecoverParams struct {
 	Signature string `json:"signature"`
 }
 
-// Manager represents account manager interface.
+// Interface represents account manager interface
+type Interface interface {
+	GetVerifiedWalletAccount(db *accounts.Database, address, password string) (*SelectedExtKey, error)
+	Sign(rpcParams SignParams, verifiedAccount *SelectedExtKey) (result types.HexBytes, err error)
+	Recover(rpcParams RecoverParams) (addr types.Address, err error)
+}
```

Comment:
This interface name doesn't describe the kind of interface, it is already declared as an interface type so this information doesn't add anything to the code.

If we look at `stdlib` interfaces they have names like `JSONMarshaller`, `Reader`, `Writer` and `Stringer`. The convention is to name the interfaces after the action they perform.

In this case the interfaces `Get`s a `VerifiedWalletAccount`, `Sign`s and `Recover`s, the name would be something that reflects this functionality. Example `AccountManager`, `KeyManager` or `KeyHandler`

Reply:
Sure, i was confused by name `Manager`, so i  thought `accounts.Interface` and `accounts.Manager` is a good pair.
And i created an issue for fixing stuff after this PR: https://github.com/status-im/status-go/issues/3625